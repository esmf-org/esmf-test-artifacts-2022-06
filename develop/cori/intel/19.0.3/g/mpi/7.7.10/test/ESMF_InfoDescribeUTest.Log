build time -- 2022-04-15 01:22:48
20220415 023858.899 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220415 023858.899 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220415 023858.899 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220415 023858.899 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220415 023858.899 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220415 023858.899 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220415 023858.899 INFO             PET0 Running with ESMF Version   : v8.3.0b12-62-gbae6f575dd
20220415 023858.899 INFO             PET0 ESMF library build date/time: "Apr 14 2022" "10:19:22"
20220415 023858.899 INFO             PET0 ESMF library build location : /global/cscratch1/sd/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_develop
20220415 023858.899 INFO             PET0 ESMF_COMM                   : mpi
20220415 023858.901 INFO             PET0 ESMF_MOAB                   : enabled
20220415 023858.901 INFO             PET0 ESMF_LAPACK                 : enabled
20220415 023858.901 INFO             PET0 ESMF_NETCDF                 : enabled
20220415 023858.901 INFO             PET0 ESMF_PNETCDF                : disabled
20220415 023858.901 INFO             PET0 ESMF_PIO                    : enabled
20220415 023858.901 INFO             PET0 ESMF_YAMLCPP                : enabled
20220415 023858.902 INFO             PET0  Beginning Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 51
20220415 023858.902 INFO             PET0  NUMBER_OF_PROCESSORS           4
20220415 023858.902 ERROR            PET0 ESMF_InfoDescribe.F90:144 ESMF_InfoDescribeInitialize() Failure  - Object already initialized
20220415 023858.902 INFO             PET0  PASS  No double initialization, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 71
20220415 023858.902 INFO             PET0  PASS  Initialize/Destroy, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 85
20220415 023858.902 INFO             PET0  PASS  Initialize behavior, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 110
20220415 023858.906 INFO             PET0  PASS  Mesh is enabled, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 146
20220415 023858.906 INFO             PET0  Ending Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 153
20220415 023858.906 INFO             PET0 Finalizing ESMF
20220415 023858.900 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220415 023858.900 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220415 023858.900 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220415 023858.900 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220415 023858.900 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220415 023858.900 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220415 023858.900 INFO             PET1 Running with ESMF Version   : v8.3.0b12-62-gbae6f575dd
20220415 023858.900 INFO             PET1 ESMF library build date/time: "Apr 14 2022" "10:19:22"
20220415 023858.900 INFO             PET1 ESMF library build location : /global/cscratch1/sd/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_develop
20220415 023858.900 INFO             PET1 ESMF_COMM                   : mpi
20220415 023858.901 INFO             PET1 ESMF_MOAB                   : enabled
20220415 023858.901 INFO             PET1 ESMF_LAPACK                 : enabled
20220415 023858.901 INFO             PET1 ESMF_NETCDF                 : enabled
20220415 023858.901 INFO             PET1 ESMF_PNETCDF                : disabled
20220415 023858.901 INFO             PET1 ESMF_PIO                    : enabled
20220415 023858.901 INFO             PET1 ESMF_YAMLCPP                : enabled
20220415 023858.902 INFO             PET1  Beginning Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 51
20220415 023858.902 INFO             PET1  NUMBER_OF_PROCESSORS           4
20220415 023858.902 ERROR            PET1 ESMF_InfoDescribe.F90:144 ESMF_InfoDescribeInitialize() Failure  - Object already initialized
20220415 023858.902 INFO             PET1  PASS  No double initialization, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 71
20220415 023858.902 INFO             PET1  PASS  Initialize/Destroy, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 85
20220415 023858.902 INFO             PET1  PASS  Initialize behavior, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 110
20220415 023858.906 INFO             PET1  PASS  Mesh is enabled, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 146
20220415 023858.906 INFO             PET1  Ending Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 153
20220415 023858.906 INFO             PET1 Finalizing ESMF
20220415 023858.899 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220415 023858.899 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220415 023858.899 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220415 023858.899 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220415 023858.899 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220415 023858.899 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220415 023858.899 INFO             PET2 Running with ESMF Version   : v8.3.0b12-62-gbae6f575dd
20220415 023858.899 INFO             PET2 ESMF library build date/time: "Apr 14 2022" "10:19:22"
20220415 023858.899 INFO             PET2 ESMF library build location : /global/cscratch1/sd/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_develop
20220415 023858.899 INFO             PET2 ESMF_COMM                   : mpi
20220415 023858.901 INFO             PET2 ESMF_MOAB                   : enabled
20220415 023858.901 INFO             PET2 ESMF_LAPACK                 : enabled
20220415 023858.901 INFO             PET2 ESMF_NETCDF                 : enabled
20220415 023858.901 INFO             PET2 ESMF_PNETCDF                : disabled
20220415 023858.901 INFO             PET2 ESMF_PIO                    : enabled
20220415 023858.901 INFO             PET2 ESMF_YAMLCPP                : enabled
20220415 023858.902 INFO             PET2  Beginning Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 51
20220415 023858.902 INFO             PET2  NUMBER_OF_PROCESSORS           4
20220415 023858.902 ERROR            PET2 ESMF_InfoDescribe.F90:144 ESMF_InfoDescribeInitialize() Failure  - Object already initialized
20220415 023858.902 INFO             PET2  PASS  No double initialization, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 71
20220415 023858.902 INFO             PET2  PASS  Initialize/Destroy, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 85
20220415 023858.902 INFO             PET2  PASS  Initialize behavior, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 110
20220415 023858.906 INFO             PET2  PASS  Mesh is enabled, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 146
20220415 023858.906 INFO             PET2  Ending Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 153
20220415 023858.906 INFO             PET2 Finalizing ESMF
20220415 023858.899 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220415 023858.899 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220415 023858.899 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220415 023858.899 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220415 023858.899 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220415 023858.899 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220415 023858.899 INFO             PET3 Running with ESMF Version   : v8.3.0b12-62-gbae6f575dd
20220415 023858.899 INFO             PET3 ESMF library build date/time: "Apr 14 2022" "10:19:22"
20220415 023858.899 INFO             PET3 ESMF library build location : /global/cscratch1/sd/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_develop
20220415 023858.899 INFO             PET3 ESMF_COMM                   : mpi
20220415 023858.901 INFO             PET3 ESMF_MOAB                   : enabled
20220415 023858.901 INFO             PET3 ESMF_LAPACK                 : enabled
20220415 023858.901 INFO             PET3 ESMF_NETCDF                 : enabled
20220415 023858.901 INFO             PET3 ESMF_PNETCDF                : disabled
20220415 023858.901 INFO             PET3 ESMF_PIO                    : enabled
20220415 023858.901 INFO             PET3 ESMF_YAMLCPP                : enabled
20220415 023858.902 INFO             PET3  Beginning Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 51
20220415 023858.902 INFO             PET3  NUMBER_OF_PROCESSORS           4
20220415 023858.902 ERROR            PET3 ESMF_InfoDescribe.F90:144 ESMF_InfoDescribeInitialize() Failure  - Object already initialized
20220415 023858.902 INFO             PET3  PASS  No double initialization, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 71
20220415 023858.902 INFO             PET3  PASS  Initialize/Destroy, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 85
20220415 023858.902 INFO             PET3  PASS  Initialize behavior, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 110
20220415 023858.906 INFO             PET3  PASS  Mesh is enabled, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 146
20220415 023858.906 INFO             PET3  Ending Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90, line 153
20220415 023858.906 INFO             PET3 Finalizing ESMF
