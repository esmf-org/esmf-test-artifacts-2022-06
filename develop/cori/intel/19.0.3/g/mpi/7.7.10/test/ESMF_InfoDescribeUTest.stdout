build time -- 2022-04-24 02:22:44
srun: Job 58172269 step creation temporarily disabled, retrying (Requested nodes are busy)
srun: Step created for job 58172269
  Beginning Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F
 90, line 51
  Beginning Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F
 90, line 51
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F
 90, line 51
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F
 90, line 51
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  PASS  No double initialization, src/Superstructure/InfoAPI/tests/ESMF_InfoDesc
 ribeUTest.F90, line 71
  PASS  No double initialization, src/Superstructure/InfoAPI/tests/ESMF_InfoDesc
 ribeUTest.F90, line 71
  PASS  No double initialization, src/Superstructure/InfoAPI/tests/ESMF_InfoDesc
 ribeUTest.F90, line 71
  PASS  No double initialization, src/Superstructure/InfoAPI/tests/ESMF_InfoDesc
 ribeUTest.F90, line 71
  PASS  Initialize/Destroy, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUT
 est.F90, line 85
  PASS  Initialize/Destroy, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUT
 est.F90, line 85
  PASS  Initialize/Destroy, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUT
 est.F90, line 85
  PASS  Initialize/Destroy, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUT
 est.F90, line 85
  PASS  Initialize behavior, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeU
 Test.F90, line 110
  PASS  Initialize behavior, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeU
 Test.F90, line 110
  PASS  Initialize behavior, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeU
 Test.F90, line 110
  PASS  Initialize behavior, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeU
 Test.F90, line 110
  PASS  Mesh is enabled, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest
 .F90, line 146
  PASS  Mesh is enabled, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest
 .F90, line 146
  Ending Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90,
  PASS  Mesh is enabled, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest
 .F90, line 146
  Ending Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90,
  line 153
  PASS  Mesh is enabled, src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest
 .F90, line 146
  Ending Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90,
  line 153
  Ending Test, file src/Superstructure/InfoAPI/tests/ESMF_InfoDescribeUTest.F90,
  line 153
  line 153
  PET           2  Test Elapsed Time    1.54200000000000       msec.
  PET           0  Test Elapsed Time    1.55500000000000       msec.
  PET           1  Test Elapsed Time    1.58500000000000       msec.
  PET           3  Test Elapsed Time    1.57099999999999       msec.
